src/main/java/com/leetcodeprep/linkedList/LinkedListCustom.java

```diff
@@ -0,0 +1,77 @@
+package com.leetcodeprep.linkedList;
+
+public class LinkedListCustom <V>{
+    private LinkedListNode head;
+    private LinkedListNode tail;
+    private int nodeCount;
+
+    public LinkedListNode add(V val){
+        if(head == null){
+            head = new LinkedListNode();
+            head.value = val;
+            tail = head;
+        }
+        else {
+            this.tail.next = new LinkedListNode();
+            this.tail.next.value = val;
+            this.tail = this.tail.next;
+        }
+        this.nodeCount++;
+        return head;
+    }
+
+    public V get(int index){
+        LinkedListNode temp = head;
+        while(index -- > 0){
+            temp = temp.next;
+        }
+        return temp.value;
+    }
+
+    public V remove(int index){
+        if(index > nodeCount || head == null){
+            throw new IndexOutOfBoundsException();
+        }
+        if(index == 0){
+            LinkedListNode t = head;
+            head = head.next;
+            t.next = null;
+            nodeCount--;
+            return t.value;
+        }
+        LinkedListNode temp = head;
+        int i=0;
+        while(i < index){
+            temp = temp.next;
+            i++;
+        }
+        LinkedListNode r = temp.next;
+        temp.next = r.next;
+        r.next = null;
+        nodeCount--;
+        return temp.next.value;
+    }
+
+    public boolean remove(V val){
+        if(head == null)
```

Comment:
Always use "{}" for each operator like "if", "while" and so on.

It helps to avoid potential issues. 
For example, you have code:
```
if(thead.next == null || thead.next.value != val)
            return false;
```
And you manager who doesn't know Java, can make very simple commit like:
```
if(thead.next == null || thead.next.value != val)
            doSomeWeirdStuff();
            return false;
```
And in this case code will work fully wrong:)